### PR TITLE
New Module: na_ontap_portset

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
- New Module: na_ontap_portset (Create and delete portset's)
- Unit Test for module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
- na_ontap_portset

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: test/units/modules/storage/netapp/test_na_ontap_portset.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+from units.compat.mock import patch
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+import ansible.module_utils.netapp as netapp_utils
+
+from ansible.modules.storage.netapp.na_ontap_portset \
+    import NetAppONTAPPortset as my_module  # module under test
+
+if not netapp_utils.has_netapp_lib():
+    pytestmark = pytest.skip('skipping as missing required netapp_lib')
```
Comment (lonico) [inline test/units/modules/storage/netapp/test_na_ontap_portset.py]: should this be:
pytestmark = pytest.mark.skip('skipping as missing required netapp_lib')

**Review by lonico — APPROVED** [review]

**Review by thedoubl3j — APPROVED** [review]
modules looks clean and simple, tests are thorough. lgtm ping @gundalow

**Comment by thedoubl3j** [comment]
cc @thaumos

**Comment by gundalow** [comment]
rebuild_merge

**Pull request merged**